packages/sc-deployer/package.json

```diff
@@ -18,7 +18,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@massalabs/as-types": "^1.1.1-dev",
+    "@massalabs/as-types": "^0.0.2-dev.20221230082918",
```

Comment:
Why we downgrade the version?

Reply:
1.1.1-dev is deprecated, latest stable version is 0.0.1, so I use latest nightly version 0.0.2

Reply:
Ok nice